tests/lib/cmsis_dsp/matrix/testcase.yaml

```diff
@@ -148,7 +148,7 @@ tests:
       - native_posix
     tags: cmsis_dsp
     min_flash: 128
-    min_ram: 64
+    min_ram: 65
```

Comment:
We know that 64k is not enough for some platforms; but, is 65k enough?

Probably not; even if it is, there would be very little headroom to work with.

My suggestion would be to increment it more leniently (e.g. by 8) to ensure that we do not need another change like this in the next few months.